dev/Prototypes/ItemsViewPrototype/FileExplorerScenario.xaml

```diff
@@ -166,9 +179,27 @@
         </StackPanel>
 
         <l:ItemsView x:Name="itemsView" 
-                     Grid.Row="1" 
+                     Grid.Row="1"
                      SelectionMode="{x:Bind ConvertToSelectionMode((x:String)((ComboBoxItem)cbSelectionMode.SelectedItem).Content), Mode=OneWay}"
                      ItemsSource="{x:Bind Items}" 
                      ViewDefinition="{StaticResource flatTableDefinition}"/>
+
+        <ScrollViewer Grid.Row="1" Grid.Column="1" Margin="10">
+            <StackPanel>
+                <TextBlock Text="Drop selected items into one of the boxes below" />
+                <Grid HorizontalAlignment="Left" Background="LightGray" AllowDrop="True" l:DragAndDropBehavior.DropCommand="{x:Bind DropCommand}" Width="100" Height="100" BorderBrush="Black" BorderThickness="1">
+                    <l:DragAndDropBehavior.Target>
+                        <l:DragAndDropPayload Data="{Binding ElementName=itemsView}" Operation="Copy" />
```

Comment:
I think the whole thing could be refactored a bit more. Perhaps a Drag behavior, and a Drop behavior, rather than a DragAndDropBehavior. But I was just following the example you had set up. The ItemsView is needed so that we can access it in the DropCommand (ItemsView.DropHandler). We need to get to the ItemsView so we can then access the selected items and figure out what to copy or move from ObservableDataSource

Reply:
The DragAndDropPayload only has 2 fields, Operation and Data. If we do not fill it out on the .Target, then we will not have the ItemsView. I know this is all very strange. I think the solution is to separate the 2 behaviors.